src/common/Footer/Footer.styled.js

```diff
@@ -0,0 +1,102 @@
+import styled from "styled-components";
+
+export const FooterContainer = styled.footer`
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  background-color: #1b2336;
+  color: ${(p) => p.theme["blue-gray-500"]};
+  padding: 50px;
+
+  .imagens {
+    display: flex;
+  }
+  h2 {
+    color: #${(p) => p.theme["secondary-900"]};
+    line-height: normal;
+    letter-spacing: -0.48px;
+  }
+`;
+
+export const Column = styled.div`
+  flex: 1;
+  padding: 20px;
+
+  &:last-child {
+    display: flex;
+    flex-direction: column;
+  }
+  .icon-link {
+    display: flex;
+    align-items: center;
+    gap: 5px;
+  }
+`;
+
+export const Logo = styled.img`
+  width: 141px;
+  height: 50px;
+  flex-shrink: 0;
+`;
+
+export const Description = styled.p`
+  display: flex;
+  width: 350px;
+  height: 90px;
+  flex-direction: column;
+  justify-content: flex-end;
+`;
+
+export const SocialIcons = styled.div`
+  display: flex;
+  justify-content: flex-start;
+  align-items: center;
+  gap: 10px;
+  padding-top: 10px;
+
+  a {
+    color: ${(p) => p.theme["blue-gray-800"]};
+    font-family: Open Sans;
+    text-decoration: none;
+    align-items: center;
+  }
+`;
+
+export const OpeningHours = styled.div`
+  p {
+    display: flex;
+    flex-direction: column;
+
+    flex-wrap: wrap;
+    color: #${(p) => p.theme["secondary-900"]};
```

Comment:
aqui podemos remover o `#`, pois na definição de tema a cor já vem com o #

```suggestion
    color: ${(p) => p.theme["secondary-900"]};
```